wire-java-generator/src/main/java/com/squareup/wire/java/JavaGenerator.java

```diff
@@ -874,9 +941,13 @@ private AnnotationSpec wireFieldAnnotation(Field field) {
   }
 
   private String adapterString(ProtoType type) {
-    return type.isScalar()
-          ? ProtoAdapter.class.getName() + '#' + type.toString().toUpperCase(Locale.US)
-          : reflectionName((ClassName) typeName(type)) + "#ADAPTER";
+    if (type.isScalar()) {
+      return ProtoAdapter.class.getName() + '#' + type.toString().toUpperCase(Locale.US);
+    } else if (protoTypeToAdapter.containsKey(type)) {
```

Comment:
Can you change this so that the `#ADAPTER` is in the map too? Otherwise I can’t do this:

```
public static final class FranklinWireAdapters {
  public static final ProtoAdapter<Money> MONEY_ADAPTER = ...
  public static final ProtoAdapter<Date> DATE_ADAPTER = ...
  public static final ProtoAdapter<Cell> CELL_ADAPTER = ...

}
```